libs/books/feature/src/lib/book-search/book-search.component.ts

```diff
@@ -13,10 +12,11 @@ import { Book } from '@tmo/shared/models';
 @Component({
   selector: 'tmo-book-search',
   templateUrl: './book-search.component.html',
-  styleUrls: ['./book-search.component.scss']
+  styleUrls: ['./book-search.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class BookSearchComponent implements OnInit {
-  books: ReadingListBook[];
+  getAllBooks$ = this.store.select(getAllBooks);
```

Comment:
I would just name this `books$` since it is an observable that contains a list of books.

Reply:
Addressed review comment